solutions/cron/pages/_app.tsx

```diff
@@ -0,0 +1,23 @@
+import type { AppProps } from 'next/app'
+import type { LayoutProps } from '@vercel/examples-ui/layout'
+import { Analytics } from '@vercel/analytics/react'
+
+import { getLayout } from '@vercel/examples-ui'
+import '@vercel/examples-ui/globals.css'
+
+function App({ Component, pageProps }: AppProps) {
+  const Layout = getLayout<LayoutProps>(Component)
+
+  return (
+    <Layout
+      title="Vercel Cron Job Example"
+      path="solutions/cron"
+      description="How to use Vercel Cron Jobs to update data at different intervals"
+    >
+      <Component {...pageProps} />
+      <Analytics />
+    </Layout>
+  )
+}
+
+export default App
```

Comment:
Is there a reason we're use `pages` instead of `app`?

Reply:
A few reasons:
- `app` is still in beta
- folks are still familiar with `pages` 
- this template is a focus on Vercel Cron

If we update this to use `app` dir we'll have to update all the examples in here to use `app` dir – I believe that should be part of a larger project

Reply:
I'm okay with keeping pages and trying to move many of these at once 👍